src/energy_systems/heat_producers/gas_boiler.jl

```diff
@@ -6,26 +6,31 @@ buffer tank and en-/disabling the boiler when a threshold is reached, in additio
 overfill shutoff condition.
 """
 mutable struct GasBoiler <: ControlledComponent
-    uac::String
-    controller::Controller
-    sys_function::SystemFunction
+    uac::String # user address code -> e.g. location and affiliation of the units within the buildings
```

Comment:
More comments on the meaning and function of code is always welcome, however I would place these in the documentation describing the general structure of a ControlledComponent implementation. This makes them universal for all other components and it helps with comments becoming stale if they are copied across components and then changes coming in.